run/test/specs/user_actions.spec.ts

```diff
@@ -135,47 +142,58 @@ async function changeUsername(platform: SupportedPlatformsType) {
       "Username is not picking up text but using access id text",
       changedUsername
     );
+  } else {
+    console.log("Username is not found`");
   }
   // select tick
-  await runOnlyOnAndroid(platform, () => device.clickOnElement("Apply"));
-  await runOnlyOnIOS(platform, () => device.clickOnElement("Done"));
+  if (platform === "android") {
+    device.clickOnByAccessibilityID("Apply");
+  } else {
+    device.clickOnByAccessibilityID("Done");
+  }
   // verify new username
 
   await closeApp(device);
 }
-
+// TO FIX (WRONG USER FLOW?)
```

Comment:
not sure what those to fix all over the place are for, but they should be fixed I guess before merge